source/content/guides/drupal-9/drupal-9-hosted-createbt/07-core.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Migrate a Site That Was Created with Build Tools to Drupal 9
+subtitle: Update Drupal Core Using Composer
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-createbt/core
+anchorid: core
+editpath: drupal-9/drupal-9-hosted-createbt/07-core.md
+---
+1. Temporarily add write access to protected files and directories:
+
+   ```bash{promptUser: user}
+   chmod 777 web/sites/default
+   find web/sites/default -name "*settings.php" -exec chmod 777 {} \;
+   find web/sites/default -name "*services.yml" -exec chmod 777 {} \;
+   ```
+
+1. Use Composer to remove `config_installer` and add new requirements:
+
+   ```bash{outputLines: 3-7,9-11}
+   composer remove drupal/config_installer --no-update
+   composer require drupal/core-recommended:^9 \
+      drupal/core-composer-scaffold:^9 \
+      drupal/core-project-message:^9 \
+      drush/drush:^10 \
+      -W --no-update
+
+   composer require phpunit/phpunit:^9 \
+      behat/behat:^3 \
+      drupal/drupal-extension:^4 \
+      --no-update -W --dev
+   ```
+
+1. If you have `core-dev` installed, follow below (skip this step if you do not have `core-dev` installed):
+
+   ```bash{outputLines: 2}
+   composer require drupal/core-dev:^9 \
+      --dev -W --no-update
+   ```
+
+1. If the **Upgrade Status** under **Reports** displays obsolete modules, update the modules using the `--no-update -W` switch to instruct Composer to check for all dependencies together rather than for each module. Replace `OBSOLETE-MODULE-NAME` in this example with the module to update:
+
+   ```bash{outputLines: 2}
+   composer require drupal/OBSOLETE-MODULE-NAME:^9 \
```

Comment:
You're now on Drupal 9. This work should have been done in the previous step when it was suggested to update the modules as upgrade status says. If you get to this point with obsolete modules, your site is probably broken and you won't be able to see upgrade status report. I'd delete this instruction

Reply:
removed